client/app/components/conference_room/layout/controls/new_event/AvailableTimeButton.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { instanceOfMoment } from 'proptypes/moment';
+import { Button } from 'react-bootstrap';
+
+const AvailableTimeButton = ({ onClick, nextEventStart }) => (
+  <Button bsStyle="primary"
+          bsSize="large"
+          onClick={() => { if (onClick) onClick(nextEventStart); }}>
```

Comment:
I did it this way because I wanted the same interface for `AvailableTimeButton` and `TimeChoiceButton`